src/tests/utils/convert.js

```diff
@@ -94,173 +105,176 @@ describe('convert render functions', () => {
   });
 
   describe('authentication', () => {
-    it('should render basic auth', (done) => {
+    it('should render basic auth', () => {
       const wbDef = definitions.basic;
 
-      convert.renderAuth(wbDef)
+      return convert.renderAuth(wbDef)
         .then(string => {
-          const auth = s2js(string);
-          auth.should.eql({
-            type: 'basic',
-            test: 'FAKE_PERFORM_FUNCTION',
-            fields: [
-              {
-                key: 'username',
-                type: 'string',
-                required: true,
-                label: 'Username'
-              },
-              {
-                key: 'password',
-                type: 'password',
-                required: true,
-                label: 'Password'
-              }
-            ],
-            connectionLabel: '{{username}}'
-          });
-          done();
-        })
-        .catch(done);
+          const auth = loadAuthModuleFromString(string);
+          auth.type.should.eql('basic');
+          auth.fields.should.eql([
+            {
+              key: 'username',
+              label: 'Username',
+              required: true,
+              type: 'string'
+            },
+            {
+              key: 'password',
+              label: 'Password',
+              required: true,
+              type: 'password'
+            }
+          ]);
+          auth.connectionLabel.should.eql('{{username}}');
+          auth.test().should.eql('./triggers/test_auth');
```

Comment:
This test trigger is mocked so that it returns the module path. See `loadAuthModuleFromString()`.